<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1011</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="user" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>821</width>
      <height>591</height>
     </rect>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <widget class="QLabel" name="label_username">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#0000ff;&quot;&gt;Label_user&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_pet1">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>160</y>
       <width>151</width>
       <height>151</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_pet2">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>160</y>
       <width>151</width>
       <height>151</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_name1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_kind1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_lv1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_xp1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_hp1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>240</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_atk1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>270</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_def1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>300</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_spd1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>330</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_name2">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>110</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_spd2">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>320</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_lv2">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>170</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_kind2">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>140</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_xp2">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>200</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_def2">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>290</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_hp2">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>230</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_atk2">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>260</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nowPet">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>330</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="levelup">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>380</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Level UP</string>
     </property>
    </widget>
    <widget class="QPushButton" name="next">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>380</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_lvUpBattle">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>450</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Promotion Fight</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_battle">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>510</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Duels Game</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>440</y>
       <width>411</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_username2">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>50</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Your opponent&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="search">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>380</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_model">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="back">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>450</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>510</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Logout</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_model_15">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>50</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="login" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>811</width>
      <height>591</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <widget class="QPushButton" name="login_button">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>330</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QLabel" name="password_label">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>250</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Password：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="username_label">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>190</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Username：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>70</y>
       <width>211</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;POKEMON GAMES&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="password">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>240</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Please input password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="regist_button">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>330</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Regist</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="name">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>180</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>Please input username</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="sendout" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>861</width>
      <height>551</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>100</y>
       <width>471</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;You failed in a battle! &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Please choose a pokemon and send  out.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_lv_1">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>240</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_xp_1">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>280</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_xp_2">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>280</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_lv_2">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>240</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_xp_3">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>280</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_lv_3">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>240</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="send1">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>360</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>This one!</string>
     </property>
    </widget>
    <widget class="QPushButton" name="send2">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>360</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>This one!</string>
     </property>
    </widget>
    <widget class="QPushButton" name="send3">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>360</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>This one!</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="userinfo" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>941</width>
      <height>561</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <widget class="QLineEdit" name="lineEdit_def">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>320</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_onlineuser">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>20</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Online users&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_name">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>80</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="next_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>470</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>NEXT</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nowPet_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>360</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_hp">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>240</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_spd">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>360</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_lv">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>160</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_atk">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>280</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lineEdit_username">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;user&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="userPetPic">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>110</y>
       <width>221</width>
       <height>211</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="openExternalLinks">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_xp">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>200</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_kind">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>120</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>60</y>
       <width>251</width>
       <height>371</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="columnCount">
      <number>9</number>
     </property>
     <column>
      <property name="text">
       <string>Username</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>WinRate</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>   Pokemon   </string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Grade</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Level</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>   ATK   </string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>   DEF   </string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>   HP    </string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>   SPD   </string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="battle">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>470</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Let's Battle!</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1011</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
